images/kubernetes-livenessprobe/configs/latest.apko.yaml

```diff
@@ -0,0 +1,33 @@
+contents:
+  repositories:
+    - /packages
+    - https://packages.wolfi.dev/os
+  keyring:
+    - /packages/local-melange.rsa.pub
```

Comment:
I think we want this to be the chainguard signing pub key?

Reply:
I don't think that we need this either